package.json

```diff
@@ -20,13 +20,16 @@
     "cz-conventional-changelog": "3.3.0",
     "husky": "^4.3.8",
     "jest": "^26.6.3",
+    "postcss": "^8.2.2",
     "prettier": "^2.2.1",
     "pretty-quick": "^3.1.0",
     "semantic-release": "^17.3.7"
   },
   "dependencies": {
-    "cssnano-preset-simple": "1.2.2",
-    "postcss": "^7.0.32"
+    "cssnano-preset-simple": "^2.0.0"
+  },
+  "peerDependencies": {
+    "postcss": "^8.2.2"
```

Comment:
v8.2.2 is needed (i think): see https://github.com/postcss/postcss/pull/1485

Reply:
https://github.com/Timer/cssnano-preset-simple/commit/42a4d113af03d526fda8cc7100c4fd768a7d9f85 is using `^8.2.8`